open-api/rest-catalog-open-api.yaml

```
@@ -541,6 +541,264 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /v1/{prefix}/namespaces/{namespace}/tables/{table}/plan:
+    parameters:
+      - $ref: '#/components/parameters/prefix'
+      - $ref: '#/components/parameters/namespace'
+      - $ref: '#/components/parameters/table'
+    post:
+      tags:
+        - Catalog API
+      summary: Submit a scan for planning
+      description:
```

```suggestion
      description: >
```
You need to add a folded block because the `-` used in the description make this technically invalid YAML.